### PR TITLE
Play 3.0 upgrade

### PR DESCRIPTION
**Pull request by davidfurey**
## What does this change?

Upgrades to Java 11, Scala 2.13, SBT 1.9.6 and Play 3.0.1

The biggest change was upgrading to the latest permissions client, since the old one didn't have a Scala 2.13 release.

NB- I also had to change the DEV application secret in SSM since [Play now enforces a minimum length of 32 characters](https://www.playframework.com/documentation/2.9.x/Migration29#Application-Secret-enforces-a-minimum-length). No change was required in CODE or PROD.

## How to test

Deploy to CODE, check app behaves as expected.

## How can we measure success?

No one notices, but Snyk vulnerabilities drop

### CODE REVIEW

**Comment by davidfurey** [comment]
Tested on CODE. I can successfully restore content to CODE composer.

File: app/AppComponents.scala [hunk 1]
```diff
@@ -20,8 +21,13 @@ class AppComponents(context: Context, identity: AppIdentity) extends BuiltInComp
...
 
-  val permissionsClient = new Permissions(config.stage)
-  logger.info(s"Permissions object initialised with config: ${permissionsClient.config}")
+  val permissionsConfig = PermissionsConfig(
+    stage = config.stage,
+    region = config.region,
+    awsCredentials = credentialsV1
+  )
+  val permissionsClient = PermissionsProvider(permissionsConfig)
+  logger.info(s"Permissions object initialised with config: $permissionsConfig")
```
Comment (twrichards) [inline app/AppComponents.scala:30]: is there a risk of this leaking secrets (now or in future if the config contains something sensitive in future)?

**Review by twrichards — APPROVED** [review]
code changes LGTM
I've tested in `CODE`, can restore just fine as before also checked removing my restore permissions and restore button is gone as expected.

File: app/AppComponents.scala (hunk 1 above)
Comment (davidfurey) [reply app/AppComponents.scala:30]: Yes. I was aiming for parity, but I can remove this

**Pull request merged**